<launch>
	<!-- declare arg to be passed in -->
	<arg name="disp" default="true"/>

	<rosparam 
		command ="load" 
		file="$(find parking)/launch/parking_parameters.yaml"/>

	<!-- H-OBCA Node -->
	<node 
		pkg="H-OBCA" 
		type="run_HOBCA.jl" 
		name="HOBCA_maneuver_server" 
		output="screen" />

	<!-- Simulation Plot Node -->
	<!-- Start this node if the disp arg is set to be true -->
	<node 
		pkg="parking" 
		type="plot_car.py" 
		name="plotCar" 
		output="screen"
		if="$(eval disp == true)"/>
	<!-- Car Control Node -->
	<node 
		pkg="parking" 
		type="car_control.py" 
		name="carNode" 
		output="screen"/>

</launch>